pdr_backend/pdr_dashboard/pages/feeds.py

```diff
@@ -264,3 +271,42 @@ def get_feeds_data_for_feeds_table(
         ]
 
         return columns, new_feed_data
+
+    def get_modal(self):
+        return dbc.Modal(
+            [
+                dbc.ModalHeader("Header"),
+                dbc.ModalBody("This is the content of the modal", id="modal_body"),
+            ],
+            id="modal",
+        )
+
+    def get_feed_graphs_modal_header(self, selected_row):
+        return html.Div(
+            html.Span(
+                f"""{selected_row["base_token"]}-{selected_row["quote_token"]}
+                {selected_row["time"]} {selected_row["exchange"]}
+                """,
+                style={"fontWeight": "bold", "fontSize": "20px"},
+            ),
+            style={
+                "display": "flex",
```

Comment:
I think we should avoid from using the style tag if it is not necessary. We can use a classname or ID and style in the css